bot/cogs/snakes.py

```diff
@@ -0,0 +1,1208 @@
+            )
+        )
+
+        embed.add_field(
+            name="Contributors",
+            value=(
+                ", ".join(contributors)
+            )
+        )
+
+        await ctx.channel.send(embed=embed)
+
+    @command(name="snakes.card()", aliases=["snakes.card"])
+    async def snake_card(self, ctx: Context, name: Snake = None):
+        """
+        Create an interesting little card from a snake!
+
+        Created by juan and Someone during the first code jam.
+        """
+
+        # Get the snake data we need
+        if not name:
+            name_obj = await self._get_snake_name()
+            name = name_obj['scientific']
+            content = await self._get_snek(name)
+
+        elif isinstance(name, dict):
+            content = name
+
+        else:
+            content = await self._get_snek(name)
+
+        # Make the card
+        async with ctx.typing():
+
+            stream = BytesIO()
+            async with async_timeout.timeout(10):
+                async with self.bot.http_session.get(content['image_list'][0]) as response:
+                    stream.write(await response.read())
+
+            stream.seek(0)
+
+            func = partial(self.generate_card, stream, content)
+            final_buffer = await self.bot.loop.run_in_executor(None, func)
+
+        # Send it!
+        await ctx.send(
+            f"A wild {content['name'].title()} appears!",
+            file=File(final_buffer, filename=content['name'].replace(" ", "") + ".png")
+        )
+
+    @command(name="snakes.fact()", aliases=["snakes.fact"])
+    async def snake_fact(self, ctx: Context):
+        """
+        Gets a snake-related fact
+
+        Written by Andrew and Prithaj.
+        Modified by lemon.
+        """
+
+        # Get a fact from the API.
+        response = await self.bot.http_session.get(self.facts_url, headers=self.headers)
+        question = await response.json()
+
+        # Build and send the embed.
+        embed = Embed(
+            title="Snake fact",
+            color=SNAKE_COLOR,
+            description=question
+        )
+        await ctx.channel.send(embed=embed)
+
+    @command(name="snakes()", aliases=["snakes"])
+    async def snake_help(self, ctx: Context):
+        """
+        This just invokes the help command on this cog.
+        """
+
+        log.debug(f"{ctx.author} requested info about the snakes cog")
+        return await ctx.invoke(self.bot.get_command("help"), "Snakes")
+
+    @command(name="snakes.snakify()", aliases=["snakes.snakify"])
+    async def snakify(self, ctx: Context, message: str = None):
+        """
+        How would I talk if I were a snake?
+        :param ctx: context
+        :param message: If this is passed, it will snakify the message.
+                        If not, it will snakify a random message from
+                        the users history.
+
+        Written by Momo and kel.
+        Modified by lemon.
+        """
+
+        def predicate(message):
```

Comment:
The simplicity of this leads me to think it could be a lambda, maybe even just a lambda passed into the `.filter()` call

Reply:
yeah okay, this one is simple enough to turn into a lambda.